tcpdialer.go

```diff
@@ -153,6 +153,9 @@ type TCPDialer struct {
 	// }
 	Resolver Resolver
 
+	// DNSCacheDuration may be used to override the default DNS cache duration (DefaultDNSCacheDuration = time.Minute)
```

Comment:
```suggestion
	// DNSCacheDuration may be used to set the DNS cache duration (default is DefaultDNSCacheDuration)
```

Reply:
I prefer not to mention the minute here as that might change in the future and will be forgotten to change here as well.

Reply:
@erikdubbelboer fixed.